src/sage/schemes/hyperelliptic_curves_smooth_model/meson.build

```diff
@@ -0,0 +1,27 @@
+py.install_sources(
+  'all.py',
+  'hyperelliptic_constructor.py',
+  'hyperelliptic_finite_field.py',
+  'hyperelliptic_g2.py',
+  'hyperelliptic_generic.py',
+  'hyperelliptic_padic_field.py',
+  'hyperelliptic_rational_field.py',
+  'invariants.py',
+  'jacobian_g2_generic.py',
+  'jacobian_g2_homset_inert.py',
+  'jacobian_g2_homset_ramified.py',
+  'jacobian_g2_homset_split.py',
+  'jacobian_generic.py',
+  'jacobian_homset_generic.py',
+  'jacobian_homset_inert.py',
+  'jacobian_homset_ramified.py',
+  'jacobian_homset_split.py',
+  'jacobian_morphism.py',
+  'mestre.py',
+  'monsky_washnitzer.py',
+  subdir: 'sage/schemes/hyperelliptic_curves_smooth_model',
+)
+
+extension_data_cpp = {}
```

Comment:
I don't think this should be here if you aren't using C++ in any Cython files.

Did you create this file manually? We have a script `tools/update-meson.py` that creates the `meson.build` files (mostly) automatically. Unless you're doing something exotic that script should "Just Work".